src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux.mdx

```diff
@@ -264,6 +264,9 @@ To install the infrastructure monitoring agent in Linux, follow these instructio
        ```
        printf "deb [arch=amd64] https://download.newrelic.com/infrastructure_agent/linux/apt buster main" | sudo tee -a /etc/apt/sources.list.d/newrelic-infra.list
        ```
+       <Callout variant="tip">
+         Use [arch=arm64] to install on ARM architectures such us AWS Graviton
```

Comment:
I'd add a period at the end of the sentence. That is: `Use [arch=arm64] to install on ARM architectures such us AWS Graviton.`